pkg/autodiscovery/listeners/kubelet.go

```diff
@@ -115,17 +115,18 @@ func (l *KubeletListener) createPodService(
 
 func (l *KubeletListener) createContainerService(
 	pod *workloadmeta.KubernetesPod,
+	podContainer workloadmeta.OrchestratorContainer,
```

Comment:
[OrchestratorContainer](https://github.com/DataDog/datadog-agent/blob/95380028e5429ff3c441bae0f6f7ea467780712c/pkg/workloadmeta/types.go#L226) seems to be a `struct`, so why not passing a pointer to this struct?